### ISSUE TITLE
Original history is missing

### ISSUE THREAD

**Issue by squimrel** [rhinstaller/isomd5sum#2]
Back in the days when this was moved out of anaconda @katzj simply copied the files out of anaconda into this repository. During that process valuable commit history got lost which even though it would require an evil force push should be added back in (in my humbly opinion).
It would be great if someone with commit access could do this by replacing the current master 
 with [squimrel/isomd5sum/tree/merged](https://github.com/squimrel/isomd5sum/tree/merged).
_The one who does this also has to check if I included any evil changes that shouldn't be there._ 
The "merged" branch is basically a continuation of [the isomd5sum anaconda history](https://github.com/squimrel/isomd5sum/tree/history) by the current [master branch](https://github.com/rhinstaller/isomd5sum/tree/master) excluding 0bd08db4f099b21ca938c174b2b650200b8a417f because it was already implemented in  9404540beabe4c3fc5c750cbeb97b1861892a87d which was lost when the project moved and also excluding the "initial import" 07cff923d3a2f4d5c5bb4b2dd47ede3ed17f305c.

In any case I'd love to hear your opinion on this. Thanks.

**Comment by bcl** [comment]
Added a history branch - https://github.com/rhinstaller/isomd5sum/tree/history
Thanks!

**Comment by squimrel** [comment]
Thank you very much! Just one question. The history branch is as up-to-date as the master branch now since you made pushed made the ["merged" branch](https://github.com/squimrel/isomd5sum/tree/merged) the history branch.
Maybe that was intentional but just as a note I also put together a ["history" branch](https://github.com/squimrel/isomd5sum/tree/history) which does not contain any commits that are in master.
The issue is that in the history branch you pushed all the commit ids are different from the actual commits so people have to be careful when referring to commits.
For example I can now refer to 1b993e6ac6db0f6cc5cb897702350d23331b057f and ask when `nread` would ever be greater than `nbytes`.
But I have to be careful if I refer to a58e7313efe4aa642e2a3fce383679cd90e1691b or 2f0df17f636232178072ec02522e3c3ca6e6dbde.
Also since those two branches have unrelated history we can not simply sync them and even though these are the "same" commits data duplication is done.
**TL;DR** IMHO it makes more sense to have ["history" branch](https://github.com/squimrel/isomd5sum/tree/history) on history instead of the merged branch.

**Comment by bcl** [comment]
I figured matching what's already there would be less confusing. I don't have any plans to continue to update it though. It's purely for historical research.
